### ISSUE TITLE
Accelerometer: support MMA7361

### ISSUE THREAD

**Issue by rwaldron** [rwaldron/johnny-five#564]
http://www.amazon.com/s/ref=nb_sb_noss?url=search-alias%3Daps&field-keywords=mma7361

**Comment by BrianGenisio** [comment]
I have one on order.  I'll knock it out when it gets here.

**Comment by BrianGenisio** [comment]
This accelerometer is not like the others :)

This unit requires a calibration step to determine the zero-values.  Unlike other accelerometers, the X,Y,Z don't necessarily have the same zero values.  The [Arduino code](https://github.com/jeroendoggen/Arduino-MMA7361-library/blob/master/AcceleroMMA7361/AcceleroMMA7361.cpp) relies on the calibrated centers of the separate axis, and I am certainly observing that they are different on my device.

Because of this, we can't use the same pattern that we have with other analog accelerometers, where we have a `zeroV` which is pre-determined and used for all three axis.  

I have a few questions/brainstorms, looking for opinions:

1. We'll need a `zeroV[]` array now.  Do we just do this in this controller?  Or do we push this down to the generic layer, and update the other controllers to accommodate?  We could allow `Number` or `Array` and use the appropriate logic based on what the controller exposes... this is probably the best approach.

2. We need to include a `calibrate` function on `Accelerometer`.  I know @rwaldron doesn't like modifying the shape for a single controller, so this means that `calibrate()` needs to be part of accelerometer now?  If so, I recommend using the same pattern as the Reflectance array.  This includes the functions `calibrate()`, `calibrateUntil(predicate)`, `loadCalibration(calibrationData)`, and `calibration` property exposed.  For the units that don't require calibration, calling `calibrate()` would overwrite the defaults.  This is probably a good thing in case some units are off from center.  I'd document in the wiki that this particular device requires a calibration.  Does this seem reasonable?

3. This unit also has a `sleep` pin which can be set to low to deactivate the device.  Do we expose the functionality for them to `enable()` and `disable()` the accelerometer?  And what about having a consistent shape?  Most of the other devices don't have this capability, but it could be a no-op or an exception in that case.

**Comment by BrianGenisio** [comment]
See submitted PR.  I took a stab at answering these questions. 

1. I am supporting `Number` or `Array` for the `zeroV` values.

2.  I am trying a more simple `autoCalibrate` option for finding centers.  It requires that the accelerometer is laying flat on the table with the Z axis pointing up.  But I expose the `zeroV` values, which can then be passed in in subsequent initializations.

3. I created an `enable(on)` function which is available to any accelerometer.  It stops responding to data.  For this device, it also puts it into sleep mode.
